src/base_keyboard_nav/navigation_controller.js

```diff
@@ -12,7 +12,8 @@
 
 import './gesture_monkey_patch';
 
-import * as Blockly from 'blockly/core';
+import {ASTNode, ShortcutRegistry} from 'blockly';
```

Comment:
I'm curious why you changed things over to use named imports. Was it to save characters?

Either way you should import from `blockly/core` not `blockly`! The `blockly` export includes the JSGenerator and bloats the package.

Reply:
I used named imports from the beginning in the code that I wrote, and changed things over in this PR for consistency across the code.

The main motivation was to save characters in code like this:
```
    ShortcutRegistry.registry.register(previousShortcut);
    ShortcutRegistry.registry.addKeyMapping(
      BlocklyUtils.KeyCodes.UP,
      previousShortcut.name,
    );
    ```

Reply:
I'll fix the imports!